spec/ttml1.xml

```diff
@@ -4303,6 +4303,7 @@ glyphs do not have an EM square that is distinct from their font's EM square.</p
 <p>If horizontal and vertical sizes are expressed independently, then the
 units of the <loc href="#style-value-length">&lt;length&gt;</loc> values must be the same.</p>
 <p>The <loc href="#style-value-length">&lt;length&gt;</loc> value(s) used to express font size must be non-negative.</p>
+<p><loc href="#style-value-length">&lt;length&gt;</loc> values specified on a <loc href="#content-vocabulary-span"><el>region</el></loc> element using <code>em</code> units are considered relative to the <emph>Computed Cell Size</emph>.</p>
```

Comment:
Are they independently related to the cell height and width, or always to the cell height? For example, is `"1em 1em"` supposed to result in the same behaviour as `"100% 100%"` or as `"100%"`?

Reply:
I think this is covered by the note:

_The expression 1c means one cell, where 'c' expresses the cell length unit as defined by 8.3.9 <length>. When a single <length> is expressed using cell units, then it refers to the height of the Computed Cell Size. When two <length> values are expressed using cell units, then the first refers to the width of the Computed Cell Size, and the second refers to the height of the Computed Cell Size._

Reply:
@palemieux that note explicitly refers to the use of cell units not `em` units so I don't think it does cover it.

Reply:
By the way, there's another relevant paragraph at https://www.w3.org/TR/ttml1/#style-value-length :

> When specified relative to a font whose size is expressed as a single length measure or as two length measures of equal length, the unit of measure em is considered to be identical to that defined by [XSL 1.1], § 5.9.13; however, when specified relative to a font whose size is expressed as two length measures of non-equal lengths, then one em is equal to the inline progression dimension of the anamorphically scaled font when used to specify lengths in the inline progression direction and equal to the block progression dimension of the scaled font when used to specify lengths in the block progression direction.

However this doesn't really help when the words "relative to a font" don't apply, as here, where the `em` measure is relative to the computed cell size.